### PR TITLE
Add: home-category pages

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  justify-content: space-between;
+  height: 4rem;
+  padding: 0 2rem;
+}
+.nav_brand_text {
+  text-transform: uppercase;
+  cursor: pointer;
+  color: var(--bg-color);
+  font-family: "Shadows Into Light", cursive;
+  font-weight: 600 !important;
```
Comment (Sahil25061999) [inline styles.css]: Avoid using of !important

**Review by Sahil25061999 — APPROVED** [review]
Ui suggestions keep the heading centered align. Great work using webp format for images code looks good the pages need a bit more styling

File: styles.css (hunk 1 above)
Comment (mehulsatardekar) [inline styles.css]: avoid using !important instead you can play with specificity. but if you want to keep it as it is you can

**Review by mehulsatardekar — APPROVED** [review]
good work @gauravsinhaweb  on naming meaning classes and semantic HTML.

**Review by JuvitaSaini09 — APPROVED** [review]
It looks good to merge

File: styles.css (hunk 1 above)
Comment (gauravsinhaweb) [reply styles.css]: Updated! Thanks!!

Comment (gauravsinhaweb) [reply styles.css]: Updated! Thanks!!

**Pull request merged**